modules/localgov_directories_location/localgov_directories_location.install

```diff
@@ -24,6 +26,15 @@ function localgov_directories_location_install($is_syncing) {
     return;
   }
 
+  $index = SearchIndex::load(Directory::DEFAULT_INDEX);
+  $location_field = new SearchIndexField($index, Directory::LOCATION_FIELD_WKT);
```

Comment:
Hello Ekes,
I am trying to understand the purpose behind this new `localgov_location_wkt` Search index field.  Is it there to avoid the [unwanted left join issue](https://github.com/localgovdrupal/localgov_directories/pull/336#issuecomment-1959483066) that Andy has pointed out?